Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -1070,44 +1023,41 @@ void EmbeddedFrag::expandEfrag(RDKit::INT_LIST &nratms,
   PRECONDITION(dp_mol, "");
 
   // first merge any fragments that share atoms in common
-  std::list<EmbeddedFrag>::iterator efri, nfri;
 
   this->mergeFragsWithComm(efrags);  //, dp_mol);
 
   while (d_attachPts.size() > 0) {
-    int aid = d_attachPts.front();
-    RDKit::INT_VECT nbrs = d_eatoms[aid].neighs;
-    CHECK_INVARIANT(nbrs.size() > 0, "");
-    RDKit::INT_VECT_I nbri;
-    RDKit::INT_LIST_I nratmi;
-    for (nbri = nbrs.begin(); nbri != nbrs.end(); nbri++) {
-      nratmi = std::find(nratms.begin(), nratms.end(), (*nbri));
+    auto aid = d_attachPts.front();
+    auto nbrs = d_eatoms[aid].neighs;
+    CHECK_INVARIANT(!nbrs.empty(), "");
+    for (auto nbri : nbrs) {
+      auto nratmi = std::find(nratms.begin(), nratms.end(), nbri);
       if (nratmi != nratms.end()) {
         // the neighbor we have to add is a non ring atoms
-        this->addNonRingAtom((*nbri), aid);  //, mol);
+        this->addNonRingAtom((nbri), aid);  //, mol);
         // remove this atom we just added from the nnratms list
         nratms.erase(nratmi);
       } else {
         // the neighbor atom must be part of a different embedded fragment -
         // merge that fragment with this one
-        nfri = efrags.end();
-        for (efri = efrags.begin(); efri != efrags.end(); efri++) {
+        auto nfri = efrags.end();
+        for (auto efri = efrags.begin(); efri != efrags.end(); ++efri) {
```

Comment:
Might as well convert to range, you're right @greglandrum, these are a bit of an eyesore now!

Reply:
same story, need the iterator later